plugins/modules/azure_rm_managementgroup_info.py

```diff
@@ -0,0 +1,329 @@
+  azure_rm_managementgroup_info:
+    name: "contoso-group"
+    children: True
+    recurse: True
+    flatten: True
+'''
+
+RETURN = '''
+management_groups:
+    description:
+        - List of Management Group dicts.
+    returned: always
+    type: list
+    contains:
+        display_name:
+            description: Management Group display name.
+            returned: always
+            type: str
+            sample: "My Management Group"
+        id:
+            description: Management Group fully qualified id.
+            returned: always
+            type: str
+            sample: "/providers/Microsoft.Management/managementGroups/group-name"
+        name:
+            description: Management Group display name.
+            returned: always
+            type: str
+            sample: group-name
+        tenant_id:
+            description: Management Group tenant id
+            returned: always
+            type: str
+            sample: "00000000-0000-0000-0000-000000000000"
+        type:
+            description: Management Group type
+            returned: always
+            type: str
+            sample: "/providers/Microsoft.Management/managementGroups"
+        children:
+            description: Child management groups or subscriptions.
+            returned: if I(children) is c(True)
+            type: list
+            sample: Nested list of children. Same as top groups, but without tenant_id.
+subscriptions:
+    description:
+        - List of subscription objects.
+    returned: if I(children) and I(flatten) are both c(True)
+    type: list
+    contains:
+        display_name:
+            description: subscription display name.
+            returned: always
+            type: str
+            sample: "some-subscription-name"
+        id:
+            description: subscription fully qualified id.
+            returned: always
+            type: str
+            sample: "/subscriptions/00000000-0000-0000-0000-feedc0ffee000000"
+        subscription_id:
+            description: subscription guid.
+            returned: always
+            type: str
+            sample: "00000000-0000-0000-0000-feedc0ffee000000"
+        type:
+            description: Management Group type
+            returned: always
+            type: str
+            sample: "/subscriptions"
+'''
+
+try:
+    from msrestazure.azure_exceptions import CloudError
+except Exception:
+    # This is handled in azure_rm_common
+    pass
+
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+
+
+class AzureRMManagementGroupInfo(AzureRMModuleBase):
+    def __init__(self):
+        self.module_arg_spec = dict(
+            children=dict(type='bool', default=False),
+            flatten=dict(type='bool', default=False),
+            id=dict(type='str'),
+            name=dict(type='str'， aliases=['management_group_name']),
```

Comment:
invalid character in identifie
```suggestion
            name=dict(type='str', aliases=['management_group_name']),
```